proposals/lra/113/implementation-document-template.adoc

```diff
@@ -0,0 +1,62 @@
+= Create new LRA programmatic client API
+:author:            Marco Sappe' Griot
+:email:             msappegr@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Requirements
+
+Create a new full REST client for the LRA Coordinator. The new client must contain methods corresponding to all of the LRA coordinator endpoints. The current client is NarayanaLRAClient which was deprecated and later moved to an internal package. The new client should replace NarayanaLRAClient by implementing its functionality. 
+NarayanaLRAClient uses the stork library to handle multiple Coordinators (a cluster of Coordinator). The new implementation must have that functionality too.
+Unit tests and integration tests are required.
+The new client must implement the MicroProfile REST Client specification:  https://microprofile.io/specifications/microprofile-rest-client/ .  
```

Comment:
Really? Do you want to pass the TCK with it?

Reply:
Thanks for your comment @xstefank , no that will not pass the TCK. I will rewrite that implementation part. I wanted to say I will use the microprofile-rest-client dependency for the client implementation.